### DISCUSSION TITLE
Welcome to Work_flow_of_population_genetics Discussions!

### DISCUSSION THREAD

**Discussion by shangshanzhizhe** [shangshanzhizhe/Work_flow_of_population_genetics#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SunWinner01** [comment]
之前有幸读过您的硕士毕业论文，今天偶然间搜到您的主页，严谨治学的师兄，好牛！

**Reply by shangshanzhizhe** [reply]
谢谢你，只是很久没有动力更新了，流程不对的话欢迎你批评指正！也欢迎催更！

**Reply by SunWinner01** [reply]
师兄晚上好~ 想问一下你之前有没有构建过SV的reference panel啊

**Reply by shangshanzhizhe** [reply]
我觉得SV reference panel 就是一个比较综合的SV鉴定结果集合吧？只要说服别人你的结果够全面，然后有全部SV的相关注释，比如影响的基因什么的，就可以称为reference panel.

**Reply by SunWinner01** [reply]
目前我准备构建SV的reference panel，用了几个不同的软件进行SV calling，哎，感觉走上了不归路，只能整合好这些结果文件，探索一下怎么构建reference panel啦... 感谢师兄回复！！！

**Comment by kunmonster** [comment]
师兄你好，在看到你关于结构变异的流程后，想咨询你一下
1. 如果我没理解错的话师兄应该是使用了Manta的joint模式 ，对吗？
2. Delly的话，您使用Delly自带的delly filter -f germlin了吗？
3. 关于变异合并，graphtyper论文中的流程是先合并每个样本使用Svimmer合并，师兄是直接对每个工具得到的群体的位点合并的吧（我也是这样）

**Reply by shangshanzhizhe** [reply]
1. 是的
2. 我没有用germline的变异过滤方法，因为我当时用的不是生殖相关的组织；germline 是人类里面常用的鉴定致病变异的方法，我当时也没用它
3. 是的！应该是不同软件的SV结果的并集

**Reply by kunmonster** [reply]
感谢回复，那我应该跟师兄的策略一样，先用每个软件的群体模式得到3个群体的位点，然后直接合并群体位点，但是请问师兄怎么处理的BND和性染色体呢？

**Reply by shangshanzhizhe** [reply]
我自己的工作中好像没有专门关注过BND，可能需要写脚本专门去提取BND的位置。具体怎么提你可以看看https://samtools.github.io/hts-specs/VCFv4.2.pdf

性染色体确实是个好问题，我之前没有做过模式生物的XY系统的性染色体，但是我想如果你有性染色体，比如Y的非同源区域信息的话，方便软件处理，可以把鉴定到的杂合基因型手动改为纯合，方便后续的参数计算和处理？我现在的工作里面是XX-XO系统，处理雄性的时候我就这么做的，X的杂合改为纯合，这里可以多考虑一下性染色体genotype的DP和常染色体的差异来设置阈值考虑是不是改为ref 纯合或者alt 纯合。我自己的工作里面好像后续的计算也没有遇到过很奇怪的问题，你可以试试，不知道能不能帮到你

**Reply by kunmonster** [reply]
谢谢师兄回复，基于这些工具都考虑了覆盖度，所以个人认为在XY系统中，如果是XY个体，那么其性染色体部分比对的时候覆盖度就预期上要少一半，但是这样很可能会影响结果（个人认为），但是基本上大部分工作中似乎都没太关注这个问题，也有可能是我多虑了，所以请教一下师兄。关于BND，几个工具出来的BND有点不一样，而且表示的意思也不太一样，看来还需要深入了解
